commands/geoadd.md

```diff
@@ -1,47 +1,87 @@
 Adds the specified geospatial items (latitude, longitude, name) to the specified
-key. Data is stored into the key as a sorted set, in a way that makes it possible to later retrieve items using a query by radius with the `GEORADIUS` or `GEORADIUSBYMEMBER` commands.
+key.
+Data is stored into the key as a sorted set, in a way that makes it possible
+to later retrieve items using a query by radius with the `GEORADIUS` or
+`GEORADIUSBYMEMBER` commands, or use `GEOSEARCH` searching within rectangular
+areas.
 
-The command takes arguments in the standard format x,y so the longitude must
-be specified before the latitude. There are limits to the coordinates that
-can be indexed: areas very near to the poles are not indexable. The exact
-limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the following:
+The command takes arguments in the standard format x,y so the longitude must be
+specified before the latitude.
+There are limits to the coordinates that can be indexed: areas very near to the
+poles are not indexable.
+The exact limits, as specified by EPSG:900913 / EPSG:3785 / OSGEO:41001 are the
+following:
```

Comment:
@kukey why did you re-format this (and other blocks in this file)?
it's hard to see if there any real changes.

Reply:
I use make, it`s auto re-format. I was suspicious at that time.

Reply:
I fix and revert the format to the old version. Thanks!

Reply:
sorry, i use rake format:cached comamnd due to this file format changed.